hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/HddsVolume.java

```diff
@@ -178,13 +189,29 @@ public File getDbParentDir() {
     return this.dbParentDir;
   }
 
+  public boolean isDbLoaded() {
+    return dbLoaded.get();
+  }
+
+  @VisibleForTesting
+  public void setTest(boolean test) {
```

Comment:
This flag is set mostly for failure tests(except for one place that I think it should be setted there).
Is it possible to mock the failure with mockito or powermock in those tests ?

Reply:
Use this flag only on DB creation decision now.